packages/keyboard-shortcuts/src/store/actions.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Returns an action object used to register a new keyboard shortcut.
+ *
+ * @param {string} name        Shortcut name.
+ * @param {string} category    Shortcut category.
+ * @param {string} description Shortcut description.
+ * @param {Object} combination Shortcut keys combination.
+ * @param {Array}  aliases     Shortcut aliases.
+ */
+export function registerShortcut( name, category, description, combination, aliases ) {
```

Comment:
Noting that we have a few different names we use for these sorts of "groupings" ("category" here). I think this one aligns reasonably well to block registration, but noting that there's another naming of "context" used for notices:

https://github.com/WordPress/gutenberg/blob/6f6412a83aaa1243ce695ed1bb7a839b7a20c75b/packages/notices/src/store/actions.js#L28-L29

Might have been something where `category` could have been used there.

Reply:
right, I think category here makes sense, ultimately (not in this PR) we should be able to register these categories with their labels... in order to group these in the keyboard shortcuts modal.